routes.go

```diff
@@ -75,6 +75,13 @@ func registerRoutes(e *echo.Echo, client *mongo.Client,dbName string) *handler.H
 	e.GET("/dev-test/slack", h.SendSlackMessage, middlewares.Authenticated)
 	e.GET("/dev-test/review", h.FetchReview, middlewares.Authenticated)
 
+	//User routes
+	e.GET("/users", h.UsersList, middlewares.Authenticated)
+	e.GET("/users/add", h.CreateUsers, middlewares.Authenticated)
+	e.POST("/add-user", h.AddUser, middlewares.Authenticated)
+	e.POST("/change-password", h.ChangePassword, middlewares.Authenticated)
+	e.POST("/edit-user", h.UpdateUser, middlewares.Authenticated)
```

Comment:
Can we be consistent with routes

instead of POST /add-user => POST /user/add
instead of POST /edit-user => POST /user/edit
instead of POST /change-password => POST /user/change-password